no_drama/__main__.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+
+import argparse
+import tempfile
+import os
+import os.path
+import shutil
+import subprocess
+import zipfile
+import glob
+import fnmatch
+
+from string import Template
+from functools import partial
+
+
+this_file = os.path.realpath(__file__)
+this_directory = os.path.dirname(this_file)
+templates_dir = os.path.join(this_directory, 'templates')
+build_lib = os.path.join(this_directory, 'build_lib')
+build_files = os.path.join(this_directory, 'build_files')
+
+
+def save_wheels(destination, packages=None, requirements_file=None,
+                find_links=None):
+    call_list = ["pip", "wheel", "--wheel-dir=%s" % destination]
+
+    if packages is not None:
+        call_list += packages
+
+    if find_links is not None:
+        call_list += ['--find-links=%s' % find_links]
+
+    if requirements_file is not None:
```

Comment:
Right.  That's 100% correct.

But, the function arguments don't NEED to default to None.

```
def save_wheels(destination, packages=None,
    requirements_file=None,
    find_links=None):
   ...
```

That's the function as is.  But, let's say I rewrote that to simply:

```
def save_wheels(destination, packages=list(),
    requirements_file=None,
    find_links=False):
```

Now, I'm being clear about what arguments I expect to my function, and I don't have to use the is None is Not None comparisons according to both PEP8 and 'Writing Idiomatic Python', I can use truthiness b/c I didn't rely on passing in a None value into my function.  It makes the code easier to comprehend for person xyz 10 years from now.